<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QWidget" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="Boton" name="boton1" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="Boton" name="boton2" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="Boton" name="boton3" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="Boton" name="boton4" native="true"/>
   </item>
   <item row="4" column="0">
    <widget class="Boton" name="boton5" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Boton</class>
   <extends>QWidget</extends>
   <header>boton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
